index.js

```diff
@@ -101,7 +109,7 @@ function makeInterceptor (spec) {
     const numInputActions = inputActions.length;
     const numOutputActions = outputActions.length;
 
-    Interceptor.attach(impl, {
+    return Interceptor.attach(impl, {
```

Comment:
Here we return an object with `detach()` instead of one with `stop()`. Might be worth adding an internal class named something like `Session` which we return in both cases, and which can hold the `listeners`.

Reply:
after some discussion offline, we've settled on the answer